rewrites.js

```diff
@@ -219,6 +219,10 @@ exports.REWRITES = [
     source: `/:collectiveSlug/:verb(donate|pay|order)/:step(${contributionFlowSteps})?`,
     destination: createOrderPage,
   },
+  {
+    source: `/:collectiveSlug/:verb(donate)/:paymentMethod(crypto)/:step(${contributionFlowSteps})?`,
+    destination: createOrderPage,
+  },
```

Comment:
Why do we need the previous one if we have this?

Reply:
You are right, we can combine both of the routes to a single one. I've done that. 👍🏽